urllib3/contrib/appengine.py

```diff
@@ -184,6 +184,7 @@ def urlopen(self, method, url, body=None, headers=None,
             retries = retries.increment(
                 method, url, response=http_response, _pool=self)
             log.info("Forced retry: %s", url)
+            retries.sleep_for_retry_after(http_response)
             retries.sleep()
```

Comment:
I think this logic will misbehave: you can sleep once for the retry after and then again for the exponential backoff. I don't think we want to do that: if we are respecting retry-after we shouldn't use our exponential backoff. The old logic here was better. =)